supplementary/alica_designer_runtime/docker-compose.yml

```diff
@@ -30,16 +31,18 @@ services:
       - SOCIAL_APP_CLIENT_ID=${SOCIAL_APP_CLIENT_ID}
       - SOCIAL_APP_SECRET=${SOCIAL_APP_SECRET}
       - OAUTH_APPLICATION_URL=http://localhost:3030
-      - NATIVE_MODE=${NATIVE_MODE:-false}
+      - NATIVE_MODE=$false
 
     command: python3 manage.py runserver 0.0.0.0:8000
     ports:
       - ${BACKEND_PORT:-9000}:8000
-    volumes:
-      - ${NATIVE_IMPORT_EXPORT_PATH:-~/alica_program}:/alica_program  # mount path for native_mode plan import/export
     depends_on:
       - db
       - redis
+    volumes:
+      - /etc/group:/etc/group:ro
+      - /etc/passwd:/etc/passwd:ro
+      - /etc/shadow:/etc/shadow:ro
```

Comment:
looks like these could be part of docker-compose.native_mode.yml

Reply:
I think it's cleaner to run the container as host user either way, and only enable/disable native mode in the docker-compose.native_mode.yml.

Reply:
Nah, people will get wary seeing this mount when there is no need to. Potential users might not be from rr and there is no way for them to know what we might do with this :-) 
Even in docker-compose.native_mode  add a comment on why this is needed.

Reply:
I have removed the volume mounts to avoid confusion. The only reason it was required was to get correct user name when inside the container which is not very useful since we only care about uid and guid